### PR TITLE
Added --format and --plain options to the 'backend.ai ps' command

### PR DESCRIPTION
**Pull request by miraliahmadli**
lablup/backend.ai#87
@achimnol 

- Added --format/-f options for "backend.ai ps"/"backend.ai admin session" commands
- Added --plain option to see process status info in plain tabular format 

- [ ] I will update the documentation file as soon as possible

### CODE REVIEW

File: docs/cli/sessions.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ other users by adding ``--access-key`` option here.
 
   backend.ai admin sessions
 
-Both commands offers options to set the status filter as follows.
+Both commands offer options to set the status filter as follows.
```
Comment (achimnol) [inline docs/cli/sessions.rst]: :+1:

File: docs/cli/sessions.rst [hunk 2]
```diff
@@ -48,6 +48,45 @@ For other options, please consult the output of ``--help``.
...
+     - ``Session ID``, ``Owner``, ``Image``, ``Type``,
+
+       ``Status``, ``Status Info``, ``Last updated``, ``Result``,
+
+       ``Tag``, ``Created At``, ``Occupied Resource``, ``Used Memory (MiB)``,
+
+       ``Max Used Memory (MiB)``, and ``CPU Using (%)``.
+
+   * - ``-f``, ``--format``
+     - ``Session ID``, ``Owner``, and specified fields by user.
```
Comment (achimnol) [inline docs/cli/sessions.rst]: I think it's better to remove these two default-included fields for full customizability.
Such "templated" options are already there: `--detail`, `--id-only`.

File: docs/cli/sessions.rst [hunk 3]
```diff
@@ -48,6 +48,45 @@ For other options, please consult the output of ``--help``.
...
+
+.. note::
+    Fields for ``-f/--format`` option can be displayed by specifying comma-separated parameters.
+
+    Available parameters for this option are: ``id``, ``status``, ``status_info``, ``created_at``, ``last_updated``, ``result``, ``image``, ``type``, ``task_id``, ``tag``, ``occupied_slots``, ``used_memory``, ``max_used_memory``, ``cpu_using``.
```
Comment (achimnol) [inline docs/cli/sessions.rst]: :+1:

File: src/ai/backend/client/cli/admin/sessions.py [hunk 4]
```diff
@@ -32,7 +51,9 @@
 @click.option('-a', '--all', is_flag=True,
               help='Display all sessions matching the condition using pagination.')
 @click.option('--detail', is_flag=True, help='Show more details using more columns.')
-def sessions(status, access_key, id_only, show_tid, dead, running, all, detail):
+@click.option('-f', '--format', default=None,  help='Display only specified fields.')
+@click.option('--plain', is_flag=True, help='Display process status in plain format.')
```
Comment (achimnol) [inline src/ai/backend/client/cli/admin/sessions.py]: "process status" looks somewhat ambiguous.
Let's say "Display the session list without decorative line drawings and the header."

Comment (achimnol) [reply src/ai/backend/client/cli/admin/sessions.py]: Also don't forget to update the same option in `ps.py`.

File: docs/cli/sessions.rst (hunk 2 above)
Comment (achimnol) [reply docs/cli/sessions.rst]: Just show an error sayng "at least one field is required for custom format" if there is no fields given at all.

File: src/ai/backend/client/cli/admin/sessions.py (hunk 4 above)
Comment (miraliahmadli) [reply src/ai/backend/client/cli/admin/sessions.py]: Got it!

File: docs/cli/sessions.rst (hunk 2 above)
Comment (miraliahmadli) [reply docs/cli/sessions.rst]: I changed that part. If user does not pass any argument, it shows error message saying that ``` Error: --format option requires an argument  ```

Comment (miraliahmadli) [reply docs/cli/sessions.rst]: Even though I print failure message "At least one field is required for custom format", it still prints the error message above. I think it is related to the ```click``` package.

**Comment by miraliahmadli** [comment]
Thank you for your feedback, @achimnol !!!

File: docs/cli/sessions.rst (hunk 2 above)
Comment (achimnol) [reply docs/cli/sessions.rst]: Ah, then it's okay to assume that there will be at least one item in the format argument.

**Pull request merged**